src/observers/DotnetLoggerObserver.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { BaseLoggerObserver } from "./BaseLoggerObserver";
+import { BaseEvent, CommandDotNetRestoreProgress, CommandDotNetRestoreSucceeded, CommandDotNetRestoreFailed } from "../omnisharp/loggingEvents";
+
+export class DotnetLoggerObserver extends BaseLoggerObserver {
+    public post = (event: BaseEvent) => {
+        switch (event.constructor.name) {
+            case CommandDotNetRestoreProgress.name:
+                this.logger.append((<CommandDotNetRestoreProgress>event).message);
+                break;
+            case CommandDotNetRestoreSucceeded.name:
+            this.logger.appendLine((<CommandDotNetRestoreSucceeded>event).message);    
```

Comment:
Is there a fallthrough issue here since `break;` is missing?

Reply:
That got missed out by mistake. Fixed that.